docs/tutorials/advanced/strategy_transformers.rst

```diff
@@ -100,6 +96,26 @@ The library includes the following transformers:
     >>> from axelrod.strategy_transformers import TrackHistoryTransformer
     >>> player = TrackHistoryTransformer(axelrod.Random)()
 
+
+Composing Transformers
+----------------------
+
+Transformers can be composed to form new composers, in two ways. You can
+simply chain together multiple transformers::
+
+    cls1 = FinalTransformer([D,D])(InitialTransformer([D,D])(axelrod.Cooperator))
+    p1 = cls1()
+
+This defines a strategy that cooperates except on the first two and last two
+rounds. Alternatively, you can make a new class using
+:code:`compose_transformers`::
+
+    cls1 = compose_transformers(FinalTransformer([D, D]),
+                                InitialTransformer([D, D]))
+    p1 = cls1(axelrod.Cooperator)()
```

Comment:
Done. There was one that I specifically left untested, the one with `*args, **kwargs`, since they would have to be defined for the test to run properly.